demo/__tests__/Translate.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { render, screen } from '@testing-library/react';
+import Translate from '../components/Translate';
+import preview from '../../dist/index';
+
+describe('Translate', () => {
+  it('render hello in English', () => {
+    render(<Translate />);
+    expect(screen.getByText('English: Hello!')).toBeInTheDocument();
+  });
+
+  it('render hello in Korean', () => {
+    render(<Translate />);
+    expect(screen.getByText('Korean: 안녕하세요')).toBeInTheDocument();
+    preview.debug();
+  });
+
+  it('render hello in Japanese', () => {
+    render(<Translate />);
+    expect(screen.getByText('Japanese: こんにちは')).toBeInTheDocument();
+    preview.debug();
+  });
+
+  it('render hello in Chinese', () => {
+    render(<Translate />);
+    expect(screen.getByText('Chinese: 你好')).toBeInTheDocument();
+    preview.debug();
+  });
+
+  it('render hello in Thai', () => {
+    render(<Translate />);
+    expect(screen.getByText('Thai: สวัสดี')).toBeInTheDocument();
+    preview.debug();
+  });
```

Comment:
Can you combine all these test cases into one to save some time running the test (avoid unnecessary renders)?. Similar to this:
```suggestion
  it('should render in multiple languages', () => {
    render(<Translate />);
    expect(screen.getByText('English: Hello!')).toBeInTheDocument();
    expect(screen.getByText('Korean: 안녕하세요')).toBeInTheDocument();
    expect(screen.getByText('Japanese: こんにちは')).toBeInTheDocument();
    expect(screen.getByText('Chinese: 你好')).toBeInTheDocument();
    expect(screen.getByText('Thai: สวัสดี')).toBeInTheDocument();
    preview.debug();
  });
```

Reply:
Sure. Fixed by https://github.com/nottyo/jest-preview/commit/9c391798b339f7fb13dab952b339102a0d8bfdcf